flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlCreateTable.java

```diff
@@ -76,11 +80,11 @@ public SqlCreateTable(
 			SqlCharStringLiteral comment) {
 		super(OPERATOR, pos, false, false);
 		this.tableName = requireNonNull(tableName, "Table name is missing");
-		this.columnList = requireNonNull(columnList, "Column list should not be null");
-		this.primaryKeyList = primaryKeyList;
-		this.uniqueKeysList = uniqueKeysList;
-		this.propertyList = propertyList;
-		this.partitionKeyList = partitionKeyList;
+		this.columnList = requireNonNull(columnList, "ColumnList should not be null");
+		this.primaryKeyList = requireNonNull(primaryKeyList, "PrimayKeyList should not be null");
```

Comment:
`ColumnList` is not a term or word or class name, it is somehow confusing with the thrown message. Same as `PrimayKeyList` and the others.

Reply:
I put the error message here because `columnList` and other fields should not be null even though they're not defined, so this is kind of a "program bug" message. Do you have a better idea on this?

Reply:
If you want to refer a field name, just use the field name withouth uppercase, i.e. `columnList`, `primaryKeyList`.

Reply:
@wuchong Thanks for pointing it out. I've fixed this.